server/build/root-module-relative-path.js

```diff
@@ -5,7 +5,7 @@
 // This function returns paths relative to the top-level 'node_modules'
 // directory found in the path. If none is found, returns the complete path.
 
-const RELATIVE_START = 'node_modules/'
+const RELATIVE_START = /[\\/]node_modules[\\/]/
```

Comment:
Shall we use [`path.sep`](https://nodejs.org/api/path.html#path_path_sep) to get the correct separator. I think that's cleaner.